src/vm/excep.cpp

```diff
@@ -7280,8 +7280,17 @@ bool IsIPInMarkedJitHelper(UINT_PTR uControlPc)
 {
     LIMITED_METHOD_CONTRACT;
 
+#ifdef _ARM_
```

Comment:
Could you please make it _ARM_ && FEATURE_PAL so that it doesn't interfere with how windows do that?